src/index.js

```diff
@@ -1 +1,32 @@
-export default () => true
+import defaultsDeep from 'lodash.defaultsdeep';
+import pickBy from 'lodash.pickby';
+
+class Stipulate {
+
+  constructor(options) {
+    this.baseOptions = options;
+  }
+
+  prefix(request) {
+    return request;
+  }
+
+  postfix(response) {
+    return response;
+  }
+
+  send(url, options) {
```

Comment:
The module isn't designed to create urls for you though. I'm not opposed to writing one that does, but I'm unconvinced it should live here. And the body of a request is definitely an `option`, so I don't really see why we should pull it out into its own parameter.